skrub/_datetime_encoder.py

```diff
@@ -294,6 +308,20 @@ def fit_transform(self, column, y=None):
             self.extracted_features_.append("total_seconds")
         if self.add_weekday:
             self.extracted_features_.append("weekday")
+        if self.add_ordinal_day:
+            self.extracted_features_.append("ordinal_day")
+
+        # Adding transformers for periodic encoding
+        self._required_transformers = {}
+
+        for _case in self.add_periodic:
+            if _case == "day":
+                _t = PeriodicEncoder(kind="spline", period=24)
+                self._required_transformers[_case] = _t
+            if _case == "year":
+                # TODO: In theory we should check that the year is leap
+                _t = PeriodicEncoder(kind="circular", period=366)
```

Comment:
isn't it a bit surprising that the encoding is different for day and year? maybe it could be splines with 4 splines or something like that?